packages/ui/package.json

```diff
@@ -25,7 +25,11 @@
     "typecheck": "tsc --noEmit --project ./tsconfig.json"
   },
   "exports": {
-    ".": "./dist-web/web.js",
+    ".": {
```

Comment:
I am not sure if we need to keep the `build:all` step above. 

We bundle into 3 folders:
- `dist-all`
- `dist-web`
- `dist-native`

Maybe we can remove the output to `dist-all` ? I'm not sure if we need it. 

We will do our imports for web from `leather-wallet/ui` and native from `leather-wallet-ui/native`.

Reply:
> Maybe we can remove the output to dist-all ? I'm not sure if we need it.

👍🏼 

> We will do our imports for web from leather-wallet/ui and native from leather-wallet-ui/native.

Why not `@leather-wallet/ui` and `@leather-wallet/ui/native`?

Reply:
I just noticed a typo in what I had said, apologies. 

It's exactly what you said: 
- Web files - `@leather-wallet/ui` 
- Native files - `@leather-wallet/ui/native`

Reply:
wondered after if it was a typo